plugin/al/usdtransaction/AL/usd/transaction/tests/CMakeLists.txt

```diff
@@ -52,13 +52,18 @@ endif()
 if(IS_MACOSX)
     # Create symbolic link to python framework
     file(CREATE_LINK ${MAYA_LOCATION}/Frameworks ${CMAKE_CURRENT_BINARY_DIR}/../Frameworks SYMBOLIC)
+    file(CREATE_LINK ${MAYA_LOCATION}/Frameworks ${AL_INSTALL_PREFIX}/Frameworks SYMBOLIC)
```

Comment:
This symbolic link was missing to be able to execute the test in both the build and install locations.